### ISSUE TITLE
Browser doesn't focus on feature tour content

### ISSUE THREAD

**Issue by juyrjola** [City-of-Helsinki/servicemap#284]
Browser should focus on the dialog generated by the feature tour. If it doesn't, screen readers do not operate well.

The problem also occurs when moving to the following feature tour panes. Browser focus should be on the instructional content of the tour instead of the service features.
